backend/services/implementations/__tests__/schoolService.test.ts

```diff
@@ -167,4 +167,20 @@ describe("mongo schoolService", (): void => {
       `School id ${notFoundId} not found`,
     );
   });
+
+  it("deleteSchool", async () => {
+    const savedSchool = await SchoolModel.create(testSchools[0]);
+
+    const deletedSchool = await schoolService.deleteSchool(savedSchool.id);
```

Comment:
```suggestion
    const deletedSchoolId = await schoolService.deleteSchool(savedSchool.id);
```
nit: variable rename